app/routes/questions/add.tsx

```diff
@@ -5,7 +5,7 @@ import {redirect} from '@remix-run/cloudflare'
 import {addQuestion, loadAllQuestions, fetchJsonList, RelatedQuestions} from '~/server-utils/stampy'
 
 const getRelated = async (question: string): Promise<RelatedQuestions> => {
-  const url = `${NLP_SEARCH_ENDPOINT}/api/search?query=${question}`
+  const url = `${NLP_SEARCH_ENDPOINT}/api/search?query=${question}&showLive=0`
```

Comment:
shouldn't these be for all questions, rather than just unpublished?

Reply:
Oops, I think I might have clobbered the work you did yesterday since I created this branch & made these changes earlier but didn't have time to continue working on this until now. I thought this function was for when the user is suggesting a new question. If that's the case, they should be requested only unpublished / non-live questions. Otherwise presumably, they should have picked a live / published answered from the previous screen. Anyway, I removed it.

Reply:
won't it need `status=all`? i.e. I forgot about this url, so it's good that you found it :D

Reply:
@mruwnik I also thought that this is for unpublished suggestions,, why do we want all, please?

Reply:
this is what gets put into the possible duplicates column of https://coda.io/d/AI-Safety-Info_dfau7sl2hmG/Incoming-questions_su7CE#_luTjm
The unpublished questions for the dialog thingy that are to be bumped, are in `app/routes/questions/search.tsx`

Reply:
There really should be some kind of central place/abstraction for all this :/

Reply:
I think the user should only see and request unpublished questions in the dialog box. What do you think, @plex? I thought the return list was getting filtered out afterward and that's why there weren't always 5 listed.

Reply:
that was the previous approach before you added filtering. Currently, there are 3 places that use search:
* the main search bar that uses tensor flow
* the dialog with unpublished questions that can be boosted -> this is in `app/routes/questions/search.tsx`
* an additional call to semantic search for all similar questions, when adding a new question -> this one here

Reply:
It's way past my bedtime and I'm delirious... I'm not fully following the difference between the last 2. So, are the results from this list ever presented to the user? Or is it just one additional check before adding it to Coda? Anyway, I have an impending work deadline, so I'll have to leave this as is for now. If you need this fixed or merged sooner, feel free to finish it off. Otherwise unfortunately, I won't be able to get back to this again until later in the week. Thanks!

Reply:
It's not even a check. The Incoming questions table in coda has a column for similar questions that are possible duplicates, to make it easier to decide how to handle new questions. This column is populated from semantic search, over the whole set of questions. Coda has a button that will go through the incoming questions and update the possible duplicates with values from semantic search. But the UI, when creating a new question, can also fill that field. So when a new question is created, the UI will additionally fetch a bunch of similar questions and set the possible duplicates directly.

This is a long-winded way of saying that no, the user will never see this list. It's all handled in the backend and only used when writing stuff to Coda

Reply:
I knew you were doing some backend magic with Coda, so I'll trust that I should just stay out of your way. Thanks for clarifying so much for though! Otherwise, I totally missed that point. Perhaps you might consider checking if the score is at least above some threshold (i.e. >0.7?) to avoid cluttering the Coda column with rather dissimilar questions?

Reply:
The threshold can be set as a parameter in the Coda formula. Currently it's 0.3, which is a bit low, but the idea was to have 5 for each item, and the reviewer can decide what to do with them.